libs/core/src/lib/card/card.component.html

```diff
@@ -1,6 +1,6 @@
 <ng-container *ngIf="!isLoading; else loaderTmpl">
     <!-- Badge -->
-    <div *ngIf="badge" class="fd-badge" tabindex="0">{{ badge }}</div>
+    <button *ngIf="badge" class="fd-badge" tabindex="0">{{ badge }}</button>
```

Comment:
It should not be a button I think. It's not clickable. Also, I don't understand for what we're having tabindex set for this element.